package.json

```diff
@@ -29,11 +29,10 @@
   "homepage": "https://github.com/CanopyTax/node-jspm-jasmine#readme",
   "devDependencies": {
     "babel-cli": "^6.6.5",
-    "babel-preset-es2015": "^6.6.0"
-  },
-  "peerDependencies": {
-    "jasmine": "*",
-    "jspm": "*"
+    "babel-preset-es2015": "^6.6.0",
+    "commander": "^2.9.0",
+    "jasmine": "^2.4.1",
+    "jspm": "^0.17.0-beta.12"
```

Comment:
Also, until there is a reason otherwise, I like the `*` for the dependency, since as far as I know node-jspm-jasmine will work with most any version of jspm and jasmine. I think it's especially useful for times like this where jspm@0.17 is in beta and some people are using it, but the majority are still on jspm@0.16.